src/containers/channelDetail/ChannelDetailContainer.jsx

```diff
@@ -14,27 +14,44 @@ import { ALL_STATUSES } from "../../constants/uiConstants";
 
 class ChannelDetailContainer extends Component {
   async componentDidMount() {
-    if (this.props.selectedSuggestion) {
-      const actualDate = new Date();
-      let oneMonthBefore = new Date().setMonth(actualDate.getMonth() - 1);
-      const oneMonthBeforeDate = new Date(oneMonthBefore);
-      let channelData = await getChannel(
-        this.props.selectedSuggestion.tokenAddress,
-        this.props.selectedSuggestion.partnerAddress
+    const selectedSuggestion = this.getSelectedSuggestion();
+    if (selectedSuggestion) {
+      if (this.props.selectedSuggestion) {
+        sessionStorage.setItem("tempSelectedSuggestion", JSON.stringify(this.props.selectedSuggestion));
```

Comment:
Careful, you shouldn't mix changes, this are changes from another PR that is not merged yet.